### ISSUE TITLE
Uncaught exception in processFrame

### ISSUE THREAD

**Issue by kalwalt** [kalwalt/webarkit-testing#11]
As said in #10 i get sometimes this error:

```
__resumeException WebARKit.js:2

  $emscripten::internal::MethodInvoker<void (WebARKit::*)(emscripten::val, ColorSpace), void, WebARKit*, emscripten::val, ColorSpace>::invoke(void (WebARKit::* const&)(emscripten::val, ColorSpace), WebARKit*, emscripten::_EM_VAL*, ColorSpace) 00d78de6:0x12cf0
  WebARKit$processFrame VM21:10
  value WebARKit.js:2
  value WebARKit.js:2
  processFrame  worker.js:53
  self.onmessage
```

Note that this happens only on Mobile device.

**Comment by kalwalt** [comment]
with commit https://github.com/kalwalt/webarkit-testing/pull/15/commits/7c4f5b22d3e83a5b50b81bd0767eadd7c766b651 i get now:

<html>
<body>
<div tabindex="-1" class="console-group console-group-messages" style="box-sizing: border-box; min-width: 0px; min-height: 0px; position: relative;"><div tabindex="-1" class="console-message-wrapper console-error-level console-selected" style="box-sizing: border-box; min-width: 0px; min-height: 0px; outline-width: 0px; display: flex; border-top-width: ; border-top-style: ; border-top-color: var(--override-error-border-color); border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: var(--override-error-border-color); --console-color-black:#000; --console-color-red:#a00; --console-color-green:#0a0; --console-color-yellow:#a50; --console-color-blue:#00a; --console-color-magenta:#a0a; --console-color-cyan:#0aa; --console-color-gray:#aaa; --console-color-darkgray:#555; --console-color-lightred:#f55; --console-color-lightgreen:#5f5; --console-color-lightyellow:#ff5; --console-color-lightblue:#55f; --console-color-ightmagenta:#f5f; --console-color-lightcyan:#5ff; --console-color-white:#fff; background-color: var(--override-focused-message-background-color); --override-error-text-color:rgb(200 0 0);"><div class="console-message" style="box-sizing: border-box; min-width: 0px; min-height: 17px; clear: right; position: relative; padding: 3px 22px 1px 0px; margin-left: 24px; flex: 1 1 auto; display: flex;"><div class="console-message-stack-trace-toggle" style="box-sizing: border-box; min-width: 0px; min-height: 0px; flex: 1 1 auto; display: flex; flex-direction: row; align-items: flex-start;"><div class="console-message-stack-trace-wrapper" aria-label="WebARKit.js:2 Uncaught 9631864 Tabella di dati sullo stack espansa" style="box-sizing: border-box; min-width: 0px; min-height: 0px; flex: 1 1 auto; display: flex; flex-direction: column; align-items: stretch;"><div class="" role="group" style="box-sizing: border-box; min-width: 0px; min-height: 0px; flex: 0 0 auto;"><span class="monospace" style="display: inline-block; --monospace-font-size:12px; --monospace-font-family:consolas,lucida console,courier new,monospace; --source-code-font-size:12px; --source-code-font-family:consolas,lucida console,courier new,monospace; box-sizing: border-box; min-width: 0px; min-height: 0px; font-family: var(--monospace-font-family); font-size: var(--monospace-font-size) !important;">

a | @ | WebARKit.js:2
-- | -- | --
  | $cv::error(int, std::__2::basic_string<char, std::__2::char_traits<char>, std::__2::allocator<char>> const&, char const*, char const*, int) | @ | 00818bc6:0x3303
  | $cv::(anonymous namespace)::SparsePyrLKOpticalFlowImpl::calc(cv::_InputArray const&, cv::_InputArray const&, cv::_InputArray const&, cv::_InputOutputArray const&, cv::_OutputArray const&, cv::_OutputArray const&) | @ | 00818bc6:0x95ae9
  | $webarkit::WebARKitAkazeTracker::track(cv::Mat&) | @ | 00818bc6:0x56615
  | $webarkit::WebARKitOrbTracker::processFrame(cv::Mat&) | @ | 00818bc6:0xab47e
  | $webarkit::WebARKitAkazeTracker::processFrameData(unsigned char*, unsigned long, unsigned long, webarkit::ColorSpace) | @ | 00818bc6:0x5773f
  | $WebARKit::processFrame(emscripten::val, webarkit::ColorSpace) | @ | 00818bc6:0x179c0a
  | $emscripten::internal::MethodInvoker<void (WebARKit::*)(emscripten::val, webarkit::ColorSpace), void, WebARKit*, emscripten::val, webarkit::ColorSpace>::invoke(void (WebARKit::* const&)(emscripten::val, webarkit::ColorSpace), WebARKit*, emscripten::_EM_VAL*, webarkit::ColorSpace) | @ | 00818bc6:0xd061d
  | WebARKit$processFrame | @ | VM24:10
  | value | @ | WebARKit.js:2
  | value | @ | WebARKit.js:2
  | processFrame | @ | worker.js:53
  | self.onmessage | @ | worker.js:16

</span></div></div></div></div></div></div><div aria-hidden="true" style="box-sizing: border-box; min-width: 0px; min-height: 0px; height: 0px; color: transparent;">﻿</div><div class="source-code" id="console-prompt" style="box-sizing: border-box; min-width: 0px; min-height: 0px; font-family: var(--source-code-font-family); white-space: pre-wrap; font-size: var(--source-code-font-size) !important; clear: right; position: relative; margin: 0px 22px 0px 20px;"><div class="widget" style="box-sizing: border-box; min-width: 0px; min-height: 0px; position: relative; flex: 1 1 auto; contain: style;"><div class="console-prompt-editor-container" style="box-sizing: border-box; min-width: 0px; min-height: 21px; color: rgb(32, 33, 36); font-family: consolas, &quot;lucida console&quot;, &quot;courier new&quot;, monospace; font-size: 12px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: pre-wrap; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><devtools-text-editor style="box-sizing: border-box; min-width: 0px; min-height: 0px;"><div class="cm-editor ͼ1 ͼ2 ͼ4 ͼ5" style="box-sizing: border-box; flex-direction: column; position: relative !important; display: flex !important; color: color: var(--color-text-primary); cursor: auto;"><div aria-live="polite" style="position: fixed; top: -10000px;"></div><div tabindex="-1" class="cm-scroller" style="font-family: var(--source-code-font-family); line-height: 1.2em; height: 22.4px; overflow-x: auto; position: relative; z-index: 0; display: flex !important; align-items: flex-start !important; font-size: var(--source-code-font-size);"><div spellcheck="false" autocorrect="off" autocapitalize="off" translate="no" contenteditable="true" class="cm-content cm-lineWrapping" role="textbox" aria-multiline="true" aria-label="Richiesta console" aria-autocomplete="list" data-language="javascript" style="margin: 0px; flex-grow: 2; flex-shrink: 1; display: block; white-space: break-spaces; overflow-wrap: anywhere; box-sizing: border-box; padding: 4px 0px; outline: none; word-break: break-word; caret-color: black; -webkit-user-modify: read-write-plaintext-only; tab-size: 4;"><div class="cm-line" style="display: block; padding: 0px 2px 0px 6px; caret-color: transparent !important;"><br></div></div><div class="cm-layer cm-layer-above cm-cursorLayer" aria-hidden="true" style="contain: size style; pointer-events: none; z-index: 150; animation-duration: 1200ms;"></div><div class="cm-layer cm-selectionLayer" aria-hidden="true" style="contain: size style; z-index: -2;"></div></div></div></devtools-text-editor></div><div class="console-eager-preview" style="box-sizing: border-box; min-width: 0px; min-height: 0px; padding-bottom: 2px; opacity: 0.6; position: relative; height: 15px; color: rgb(32, 33, 36); font-family: consolas, &quot;lucida console&quot;, &quot;courier new&quot;, monospace; font-size: 12px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: pre-wrap; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><br class="Apple-interchange-newline">
</body>
</html>

I think i have to change the initialization of the trackers.

**Comment by kalwalt** [comment]
i tested removing the m_orb_tracker and all related code and seems that the uncaught error does not show anymore. I will try to change the code in accord of this in dedicated PR.

**Comment by kalwalt** [comment]
This issue should be fixed with #16
